release_notes.md

```diff
@@ -30,8 +60,6 @@ permalink: /release_notes/
 * Primarily bug fixes. SBR rules now working again on most maps (map XMLs updated, download of latest maps required)
 
 
-## Changes for 1.9.0.0
```

Comment:
Why is this title being removed?

Reply:
Bumping for @simon33-2.

Reply:
Looks like a meaningless line. Perhaps a typo but I don't think it matters much either way.